.github/workflows/ci.yaml

```
@@ -3,6 +3,21 @@ on:
   push:
     branches:
       - master
+    paths:
+      - .github/workflows/ci.yaml
+      - augur/
+      - docs/
+
+      # Used for testing
+      - bin/
+      - scripts/
+      - tests/
+      - .coveragerc
+      - .cramrc
+      - .flake8
+      - codecov.yml
+      - mypy.ini
+      - pytest.python3.ini
```

What about:
- .pylintrc could potentially cause linting to fail when we change it?
- .readtehdocs.yaml could change docs settings?
- setup.py changes what `pip install .[dev]` installs no?
```suggestion
      - pytest.python3.ini
      - .pylintrc
      - .readthedocs.yml
      - setup.py
```
```

---

Good point - I'll take a look after lunch

---

I thought about it some more and don't think this list is worth maintaining just to save a few unnecessary CI runs. I think doing this properly would be something more like 0dc57e6e3b0c7badf53c37dcdd970df5f289644d instead of 9c85aab60e467fd57b4b0ad04d2624db44f3364e (what you had originally reviewed). I'm not satisfied with either of those and thinking it's better to remove from this PR.

---

Good point, still not wasted work if it made us realize the best way forward :)